release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -297,6 +297,14 @@ The following related APIs will be removed in a future release:
 [id="ocp-4-3-bug-fixes"]
 == Bug fixes
 
+*Web Console*
+
+* Previously, for admin users, non-serverless workloads showed resources for serverless workloads in the *Topology* resources panel and non-admin users would not be able to view any workloads. With this bug fix, the resources panel works as expected in the case of normal and Knative specific deployments and a non-admin user can now view the workloads.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1758628[*BZ#1758628*])
+* Previously, Pod status was depicted inconsistently in the *Topology* view, *Resources* side panel, and the *Deployment Config Details* page. With this bug fix, the *Topology* and *Resources* pages share the same logic thus depicting the Pod status consistently.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1760827[*BZ#1760827*])
+* Previously, when an application was created using the *Add* page options, the deployed image ignored the selected target port and always used the first entry. With this bug fix, code in the `import-submit-utis` and `deployImage-submit-utils` files has been refactored to use shared code, enabling the deployed image to use the selected port.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1760836[*BZ#1760836*])
```

Comment:
Should `*Add* page options` be `*Add page* options`?

Reply:
No, it is just Add.

Reply:
So, should it read `*Add* option` (or `*Add* page option)?